test/Elastic.Apm.Tests/BackendCommTests/PayloadSenderTests.cs

```diff
@@ -348,6 +348,20 @@ internal async void FlushInterval_test(TestArgs args, int numberOfEventsToSend)
 			}
 		}
 
+		[Fact]
+		public void Find_a_better_name()
+		{
+			var logger = new InMemoryBlockingLogger(LogLevel.Warning);
+			var configReader = new MockConfiguration(logger, apiKey: "*** INVALID_API_KEY ***");
+			var mockHttpMessageHandler = new MockHttpMessageHandler((r, c) => Task.FromResult(new HttpResponseMessage(HttpStatusCode.OK)));
+			var service = Service.GetDefaultService(configReader, logger);
+
+			using (new PayloadSenderV2(logger, configReader, service, new Api.System(), null, mockHttpMessageHandler, TestDisplayName))
+			{
+				logger.Lines.Should().Contain(l => l.Contains("Failed parsing APM Server version - version string not available"));
```

Comment:
It should succeed as long as *one* log line matches the expected pattern, so I guess the expected log line was not present in that case at all.
That being said, the test method was updated to specifically test `ApmServerInfoProvider` and moved into a dedicated test class.

Reply:
> It should succeed as long as one log line matches the expected pattern, so I guess the expected log line was not present in that case at all.

Yeah, that was it - because we had no content in the HTTP call, it never reached the log line we assert on here.